spec/requests/media_auth_request_spec.rb

```diff
@@ -2,17 +2,11 @@
 
 RSpec.describe "Authentication for Media requests", type: :request do
 
-  let(:allowed_loc) { 'ip.address1' }
   let(:user_no_loc_no_webauth) { User.new }
-  let(:user_no_loc_webauth) { User.new(webauth_user: true) }
```

Comment:
The CanCan ability for non-stanford WebAuth users is tested here: https://github.com/sul-dlss/digital_stacks_rails/blob/92-remove-media-download/spec/models/user_spec.rb#L96-L151 (bonus stream vs. download tests all over the place in that file).

This action is an implementation of `can? :stream, StacksMediaStream`.  I feel like the various user/rights scenarios are tested thoroughly between the spec above and the [`FileAuthRequestSpec`](https://github.com/sul-dlss/digital_stacks_rails/blob/92-remove-media-download/spec/requests/file_auth_request_spec.rb).

This test is doing some high-level tests to execute the code paths of 1) if a user can stream and 2) if a user can't stream.  The details of what makes a user be able to stream or not is tested elsewhere.  Also, within the `user can't stream` conditional branch, I feel that the `MediaAuthenticationJSON` thoroughly tests the scenarios of different user and file objects.

I would pose this question:  If we add a new file type with its own controller, and it implemented an action that checked that a user could read a file (which is already thoroughly tested), would we then need to setup every permutation of user and rights to test the results of that action?  That would seem unnecessary to me (and potentially be an undue increase in maintainability burden).

That being said, I would be more than happy to sit down with you tomorrow, and maybe we can hunt down some scenarios that I've missed in this refactor and we can make sure that they are tested appropriately.